frontend/app/components/inspectionsList.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+import PropTypes from 'prop-types';
+import { ListGroup, ListGroupItem, Panel } from 'react-bootstrap';
+
+/*
+class BSTable extends React.Component {
+  render() {
+    const { data } = this.props
+    if (this.props.data) {
+      return (
+        <BootstrapTable data={data} keyField='inspection_date'>
+          <TableHeaderColumn dataField='violation_description'>Violation Description</TableHeaderColumn>
+          <TableHeaderColumn dataField='inspection_date'>Inspection Date</TableHeaderColumn>
+        </BootstrapTable>);
+    } else {
+      return (<p>?</p>);
+    }
+  }
+}
+*/
+
+class InspectionsList extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  isExpandableRow(row) {
+    if (row.business > 2) return true;
+      else return false;
+  }
+
+  expandComponent(row) {
+    return (
+      <div>
+        <ListGroup>
+          <ListGroupItem header="Inspection Type">
+             {row['inspection_type']}
+          </ListGroupItem>
+          <ListGroupItem header="Action">
+            {row['action']}
+          </ListGroupItem>
+          <ListGroupItem header="Violation Description">
+            {row['violation_description']}
+          </ListGroupItem>
+          <ListGroupItem header="Violation Code">
+            {row['violation_code']}
+          </ListGroupItem>
+          <ListGroupItem header="Critical">
+            {row['critical_flag']}
+          </ListGroupItem>
+          <ListGroupItem header="Grade Date">
+            {row['grade_date'] || 'N/A'}
+          </ListGroupItem>
+        </ListGroup>
+      </div>
+    )
+  }
+
+  render() {
+    const { inspections } = this.props
+    if (this.props.inspections) {
+      return (
+          <div className="container">
+            <div className="row">
+              <div className="col-lg-12 col-xs-12">
+              <Panel header="Inspections">
+              <BootstrapTable
+                data={inspections}
+                expandColumnOptions={ { expandColumnVisible: true } }
```

Comment:
the default +/- that are there for the expanding rows are ugly, but they are changeable.